charts/kvisor/values-local.yaml

```diff
@@ -21,10 +21,11 @@ agent:
     signature-socks5-detection-enabled: true
     container-stats-enabled: true
     ebpf-events-enabled: true
-    netflow-enabled: true
+    netflow-enabled: false
```

Comment:
Why is it disabled?

Reply:
This was a left-over from playing around (it did fill up my local db quite quickly, haha).
Will switch back to true.